packages/components/src/List/ListComposition/Manager/ListManager.component.js

```diff
@@ -50,7 +54,7 @@ function Manager({
 		setSortParams,
 		setTextFilter,
 		setColumns,
-		setVisibleColumns,
+		setVisibleColumns: nextVisibleColumns => setVisibleColumns(columns, nextVisibleColumns),
```

Comment:
setVisibleColumns could be exposed by the custom hook, it would simplify integration and API 💭